modules/ROOT/pages/about-api-catalog-cli.adoc

```diff
@@ -0,0 +1,718 @@
+
+== Create a Descriptor File Using the CLI
+
+Before you run the commands to publish your APIs, you must create a descriptor file that has the information Exchange needs to catalog the APIs. You can create the descriptor file using the CLI or you can create it manually. Typically you should create the file using the CLI and if you need to set more options to control which APIs are published, you can update the file manually. 
+
+You can run the `api-catalog create-descriptor` command to create and print the catalog descriptor YAML results to a designated file. The command will output cataloging information for all API specifications it finds in the full directory tree relative to the current working directory.
+
+To create a descriptor file using the CLI:
+
+. Navigate to the directory path that contains the APIs you want to catalog. 
+. Run the `api-catalog create-descriptor` command. The following example creates a descriptor file called `mydescriptor.yaml` in the current path:
++
+----
+> api-catalog create-descriptor -f mydescriptor.yaml
+----
+
+The result of the command depends on the presence and content of the designated descriptor file as follows:
+
+* No file or empty file: The command will discover the APIs and write the results to standard output and to a file named `catalog.yaml`.
+
+* Existing file with content: The command will fail and show an error message that says the descriptor file already exists.
+
+NOTE: You can alternatively create a descriptor file manually instead of using the `api-catalog create-descriptor` command to generate the file. See <<Create or Update a Descriptor File Manually>>.
+
+==== api-catalog create-descriptor
+
+Following is a reference for the `api-catalog create-descriptor` command. 
+
+*Usage*
+
+----
+> api-catalog create-descriptor [options] 
+----
+
+*Options*
+
+`-f, --file=file`
+  +
+  [default: `catalog.yaml`] 
+  + 
+  The name and location in which to save the generated catalog descriptor file.
+
+== Publish APIs Using the CLI
+
+You can use the `api-catalog publish-asset` command to publish your APIs to Exchange. You will typically embed this command in your automation tools, such as a CI/CD pipeline or custom scripts, to automatically trigger the cataloging of your API assets. The command publishes your APIs using the information in the file specified in the `--descriptor-file` option.   
+
+=== Prerequisites
+
+Create a descriptor file that describes the APIs to publish and copy it to the file path where your API projects are stored. See <<Create a Descriptor File Using the CLI>>.
+
+=== Limitations
+
+This version of API Catalog CLI has the following limitations:
+
+* Documentation must be in Markdown format.
+* Images in documentation are resolved only if the references are full URLs that point to externally hosted images. 
+* Links between documentation pages are resolved only if the references are full URLs that point to externally hosted documentation.
```

Comment:
I would remove the `between documentation pages` since we do not support linking between documentation pages. We only support links to external resources.

Reply:
Done! Thank you!